zeton/templates/index_child.html

```diff
@@ -15,14 +15,16 @@
             <button>LOGOUT</button>
         </a></div>
     <div class="rectangle child">
-        <h2>Witaj {{ firstname }}, obecnie masz {{ points }} punktów</h2>
+        <h2>Witaj {{ child.firstname  }}, obecnie masz {{ child.points }} punktów</h2>
         <h3>Szkolny rekord tygodnia : {{ school_weekly_highscore }}</h3>
-        {% if ban %}
-            <h2 style="text-align: center; color: red;"> Masz BANA !</h2>
-            <h4> koniec bana za {{ time_ban_stop }} minuty</h4>
+
+        {% if child.bans.6.active %}
+        <h2 style="text-align: center; color: red;"> Masz BANA !</h2>
+        <h4> Ban kończy się: {{ child.bans.6.stop }}</h4>
         {% else %}
-            <h2 style="color: green;">Możesz korzystać z komputera</h2>
+        <h2 style="color: green;">Możesz korzystać z komputera</h2>
```

Comment:
Tu widzę błąd, bo jeśli mam 'BAN 2 stopnia na 30 min - brak komputera i telefonu' to nie mogę korzystać z komputera.

Proponuję zamienić blok if na:

        {% if child.bans.4.active or child.bans.6.active %}
            {% if child.bans.4.active %}
                <h2 style="text-align: center;"> Masz BANA !</h2>
                <h4> Ban kończy się: {{ child.bans.4.stop }}</h4>
            {% endif %}
            {% if child.bans.6.active %}
                <h2 style="text-align: center; color: red;"> Masz BANA !</h2>
                <h4> Ban kończy się: {{ child.bans.6.stop }}</h4>
            {% endif %}
        {% else %}
                <h2 style="color: green;">Możesz korzystać z komputera</h2>
        {% endif %}

Reply:
To też nie jest tak jakbym chciał. W Twoim proponowanym rozwiązaniu wyświetlą się dwa napisy" "Masz Bana" w momencie gdy oba będą aktywne. Powielamy informację z bloku konsekwencji. Dodatkowo inne dzieci mają różne bany np: ban na tablet i telewizje. Proponuje wystawić nowy issue na zmianę nagłówka strony - tak aby wyświetlał jedynie nawyższy aktywny warn/ban dla danego dziecka.

Reply:
Aha no tak, racja.